app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>This is the home page</h1>
\ No newline at end of file
+<h1>List of all users</h1>
+
+<table>
+  <tr>
+    <th><b>Photo</b></th>
+    <th><b>Username</b></th>
+    <th><b>No of posts</b></th>
+  </tr>
+  <% @users.each do |user| %>
+    <tr>
+      <td><img src="<%= user.photo %>" alt="User Photo"></td>
+      <td><%= user.name %></td>
+      <td><%= user.posts_counter %></td>
+      <td><%= link_to 'User', user_path(user.id) %></td>
+    </tr>
+  <% end %>
```

Comment:
| Your Design | Given Design |
| --- | --- |
| ![image](https://github.com/Cabraham1/Blog/assets/93039359/c3acbf28-9e71-473b-9407-291e53306e79) | ![image](https://github.com/Cabraham1/Blog/assets/93039359/148cbd93-15a0-4bdf-a028-5c9dc6acc89d) |

- [ ] Great work on adding the image 👍 but I guess it would be great if you follow the wireframe as suggested by the previous reviewer that your image should be of the same size as the details are which you can see in the wireframe 😊

- [ ] As suggested by the previous reviewer make all your pages look like the given wireframe 👍 so, It would be nice if you organize your user details because if you compare your design with the given design then you would find a lot of things such s padding, margins, spaces, etc which are not similar 🤔 Kindly make these changes to make your project follow the given design 👍